projects/packages/jetpack-mu-wpcom/src/features/wpcom-global-styles/index.php

```diff
@@ -598,39 +615,64 @@ function wpcom_display_global_styles_launch_bar() {
+							>
+								<?php esc_html_e( 'Upgrade and keep premium styles', 'jetpack-mu-wpcom' ); ?>
+							</a>
+							<?php
+						} else {
+							?>
+							<a
+								class="launch-bar-global-styles__button--upgrade"
+								href="<?php echo esc_url( $upgrade_url ); ?>"
+							>
+								<?php esc_html_e( 'Upgrade to unlock premium styles', 'jetpack-mu-wpcom' ); ?>
+							</a>
+							<?php
+						}
+						?>
 						<a
-							class="launch-bar-global-styles-upgrade"
-							href="<?php echo esc_url( $upgrade_url ); ?>"
-						>
-							<?php echo esc_html__( 'Upgrade now', 'jetpack-mu-wpcom' ); ?>
-						</a>
-						<a
-							class="launch-bar-global-styles-reset"
-							href="https://wordpress.com/support/using-styles/#reset-all-styles"
-							target="_blank"
-						>
-							<svg width="15" height="14" viewBox="0 0 15 14" fill="none" xmlns="http://www.w3.org/2000/svg">
-								<path d="M5.8125 5.6875C5.8125 4.75552 6.56802 4 7.5 4C8.43198 4 9.1875 4.75552 9.1875 5.6875C9.1875 6.55621 8.53108 7.2716 7.6872 7.36473C7.58427 7.37609 7.5 7.45895 7.5 7.5625V8.5M7.5 9.25V10.375M13.5 7C13.5 10.3137 10.8137 13 7.5 13C4.18629 13 1.5 10.3137 1.5 7C1.5 3.68629 4.18629 1 7.5 1C10.8137 1 13.5 3.68629 13.5 7Z" stroke="#1E1E1E" stroke-width="1.5"/>
-							</svg>
-							<?php echo esc_html__( 'Remove premium styles', 'jetpack-mu-wpcom' ); ?>
-							<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="15" height="15" aria-hidden="true" focusable="false"><path d="M18.2 17c0 .7-.6 1.2-1.2 1.2H7c-.7 0-1.2-.6-1.2-1.2V7c0-.7.6-1.2 1.2-1.2h3.2V4.2H7C5.5 4.2 4.2 5.5 4.2 7v10c0 1.5 1.2 2.8 2.8 2.8h10c1.5 0 2.8-1.2 2.8-2.8v-3.6h-1.5V17zM14.9 3v1.5h3.7l-6.4 6.4 1.1 1.1 6.4-6.4v3.7h1.5V3h-6.3z"></path></svg>
-						</a>
-						<a class="launch-bar-global-styles-preview" href="<?php echo esc_url( $preview_location ); ?>">
-							<label><input type="checkbox" <?php echo wpcom_is_previewing_global_styles() ? 'checked' : ''; ?>><span></span></label>
-							<?php echo esc_html__( 'Preview premium styles', 'jetpack-mu-wpcom' ); ?>
-						</a>
+							class="launch-bar-global-styles__button--reset"
+								href="https://wordpress.com/support/using-styles/#reset-all-styles"
+								target="_blank"
+								data-blog-id="<?php echo esc_attr( (string) $blog_id ); ?>"
+								data-global-styles-id="<?php echo esc_attr( (string) $global_styles_id ); ?>"
+							>
+								<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="18" height="18" aria-hidden="true" focusable="false"><path fill-rule="evenodd" clip-rule="evenodd" d="M12 5.5A2.25 2.25 0 0 0 9.878 7h4.244A2.251 2.251 0 0 0 12 5.5ZM12 4a3.751 3.751 0 0 0-3.675 3H5v1.5h1.27l.818 8.997a2.75 2.75 0 0 0 2.739 2.501h4.347a2.75 2.75 0 0 0 2.738-2.5L17.73 8.5H19V7h-3.325A3.751 3.751 0 0 0 12 4Zm4.224 4.5H7.776l.806 8.861a1.25 1.25 0 0 0 1.245 1.137h4.347a1.25 1.25 0 0 0 1.245-1.137l.805-8.861Z"></path></svg>
+								<?php esc_html_e( 'Remove premium styles', 'jetpack-mu-wpcom' ); ?>
```

Comment:
Not sure if it's an artifact of the sandboxing but I can't get this link to work in testing.

Reply:
You probably need to enable write mode?